cartridge-cli.lua

```diff
@@ -3591,25 +3600,6 @@ end
 local function collect_processes(args)
     local processes = {}
 
-    -- get instance names
-    local instance_names
-    if args.instance_name == nil then
-        local err
-        instance_names, err = get_configured_instances(args.cfg, args.app_name)
-        if instance_names == nil then return nil, err end
-    else
-        instance_names = {args.instance_name}
-    end
-
-    -- instances
-    for _, instance_name in pairs(instance_names) do
-        local instance_args = table.copy(args)
-        instance_args.instance_name = instance_name
-
-        local process_args = get_process_args(instance_args)
-        table.insert(processes, process_args)
-    end
-
     -- stateboard
     if args.stateboard then
```

Comment:
`--with-stateboard`

Reply:
`args.stateboard = args.stateboard or args.stateboard_only`, this flag will be set to prevent missing `args.stateboard or args.stateboard_only` everywhere in the code.

Reply:
This has nothing to do with my remark. However, as well as my remark to your patch.